falcon_kit/util/system.py

```diff
@@ -18,10 +18,10 @@ def only_these_symlinks(dir2paths):
         base2rel = {os.path.basename(path): os.path.relpath(path, d) for path in paths}
         assert len(base2rel) == len(bases), 'Non-unique basename in {}'.format(repr(paths))
         for existing_base in os.listdir(d):
-            existing_path = os.path.join(d, existing_basename)
+            existing_path = os.path.join(d, existing_base)
```

Comment:
Actually, this whole PR is a tiny change, but with several commits. Now is a good time to learn how to squash with interactive rebase:
```sh
git checkout -B minor_type_fix
git pull --rebase origin/master
git rebase -i origin/master
```
Then, in your editor, change `pick` to `squash`. But keep the **first** `pick`. Then save and exit. Your squashed commits will become part of the the picked commit.

Try it! If you screw up, no problem. Everything can be undone in git!